### ISSUE TITLE
Progress Dots Not Changing Colors

### ISSUE THREAD

**Issue by macdadmin** [Shufflepuck/SplashBuddy#120]
I am using this internally and it works great.  The little dots next to the action go from gray to amber to green and then moves on.  I have also set this up for a client and is working equally great, however the dots are not changing colors.  Anyone seen this before?  Anything I can do to fix it?  Thanks in advance.

**Comment by cybertunnel** [comment]
Are you making sure the packages are using this Scheme: `Package Name-1.2.3.pkg`? Where `Package Name` SplashBuddy's preferences would be looking for the whole "Package Name"?
If not using a package, are you using either ghost package (empty package) or a script to write that the "package" was installed?

**Comment by macdadmin** [comment]
As far as I can tell… I’ve followed the steps accordingly.  I took our functional setup that we have running internally (dots light up), and duped the folder and redid the config for the plist, icons and the HTML file.  Everything works great.  Just can’t get the lights to light up.

Let me know if you have any suggestions.  I’m going to go through the config again today.  I have rebuilt it twice.

Thanks.

Adam
